config/features.yml

```diff
@@ -16,12 +16,6 @@ features:
     actor_type: user
     description: Use lighthouse instead of EVSS to view/download benefit letters.
     enable_in_development: true
-  binding_flow_modal_dsl:
-    actor_type: user
-    description: Enables grandfathering modal to begin binding from dslogon to login.gov
-  binding_flow_modal_mhv:
-    actor_type: user
-    description: Enables grandfathering modal to begin binding from mhv to login.gov
```

Comment:
Our feature flipper cleanup job is having some issues, so someone might need to go into the rails console and run `Flipper.remove("name_of_feature_flag")` on these. If no one on your team has that access, I (or someone on Support) can do it for you.

Reply:
I can do that on staging, but not production. Would you mind handling the clean up on the production side?

Reply:
Done on staging
```
irb(main):001:0> Flipper.remove("binding_flow_modal_dsl")
=> true
irb(main):002:0> Flipper.remove("binding_flow_modal_mhv")
=> true
```